app/bundles/CoreBundle/Helper/DateRelativeParser.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
```

Comment:
```suggestion
 * @copyright   2019 Mautic Contributors. All rights reserved
```
0r 2018. I don't think it was 2014 :)